docs_src/query_usb_camera.md

```diff
@@ -1,25 +1,25 @@
-# How to query a usb camera
+# Query USB Camera
```

Comment:
What is the need to query a USB camera? This page is not linked anywhere. How will the user reach this page?

Reply:
Let me verify with the team but we may be able to remove this document.

Reply:
somehow doc evolves and it was gone. I just added the comment on L37 of pipelinerun.md, that's where it should have link to query USB Camera